src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasInMemoryService.kt

```diff
@@ -83,4 +87,23 @@ class CosmasInMemoryService : CosmasGrpc.CosmasImplBase() {
             this.files[request.fileId] = fileVersions
         }
     }
+
+    override fun createPatch(request: CosmasProto.CreatePatchRequest,
+                                        responseObserver: StreamObserver<CosmasProto.CreatePatchResponse>) {

```

Comment:
Please align this argument. Either left-align with the previous one:
```
override fun createPatch(request: CosmasProto.CreatePatchRequest,
                         responseObserver: StreamObserver<CosmasProto.CreatePatchResponse>) {
```

or 4 spaces indent relative to the first column of the previous line 
```
override fun createPatch(request: CosmasProto.CreatePatchRequest,
    responseObserver: StreamObserver<CosmasProto.CreatePatchResponse>) {
```